src/pkg/packager/load/load.go

```diff
@@ -27,6 +28,9 @@ import (
 type DefinitionOptions struct {
 	Flavor       string
 	SetVariables map[string]string
+	// SkipRequiredValues ignores values schema validation errors when a "required" field is empty. Used when a package
+	// value should be supplied at deploy-time and doesn't have a default set in the package values.
+	SkipRequiredValues bool
```

Comment:
Skeleton package publishes should also set this to true

Reply:
added to PublishSkeleton's load.DefinitionOptions